pom.xml

```diff
@@ -203,5 +203,17 @@
             <version>${hamcrest.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
make use of powermock to test some implementations (private and Activator.class, and various  static methods)